stories/settings.stories.js

```diff
@@ -82,13 +83,13 @@ storiesOf( 'Settings', module )
 			provideModuleRegistrations( registry );
 		};
 
+		const history = createHashHistory();
+
 		return (
 			<WithTestRegistry callback={ setupRegistry } >
-				<div className="mdc-layout-grid__inner">
-					<div className="mdc-layout-grid__cell mdc-layout-grid__cell--span-12">
-						<SettingsActiveModules />
-					</div>
-				</div>
+				<Router history={ history }>
```

Comment:
If we're applying the Router in the `Root` we should probably add it with a global decorator in `preview.js`. If needed, an alternate `history` could be provided via `parameters` similar to `features`.

Reply:
It already exists in https://github.com/google/site-kit-wp/blob/9d9c1c3a57455927c9e5773eb51cb7df2630f4ca/tests/js/utils.js#L105, but I've moved it from `<Root>` for now anyway, so I think this is better for the time being.